content-src/components/Routes/Routes.js

```diff
@@ -1,31 +1,31 @@
 const React = require("react");
```

Comment:
Can you add this stuff to the `Base` file instead and use that instead of `Routes`? That seems like it makes more sense